### DISCUSSION TITLE
Animação em SVG interativa com o mouse

### DISCUSSION THREAD

**Discussion by DouglpDesign** [frontendbr/forum#1731] [Perguntas e Respostas]
Olá, quais conhecimentos preciso aprender ou conteúdo que devo procurar para desenvolver animações interativas assim? Cheguei a fazer uma no Adobe animate mas não sei criar essa fluidez e colocar a interação do mouse.

https://www.youtube.com/watch?v=G2NoeG0mXzg

**Comment by giovannism20** [comment]
Olá Douglas, bom você precisa aprender a manipular SVG (normalmente usamos JavaScript).

Veja se esse vídeo lhe ajuda, qualquer dúvida só mandar mensagem:

[Animando SVG](https://www.youtube.com/watch?v=PnMTadIWf-Q)

**Comment by brunogarciavaz** [comment]
No exemplo do vídeo com certeza está sendo usado um canvas, e não um SVG animado.
Encontrei este codepen com um efeito parecido, pode te colocar na direção certa: https://codepen.io/shubniggurath/pen/EmMzpp
